### PR TITLE
[US170 Task185] Adding API Gateway

### PR DESCRIPTION
**Pull request by achaur11**
API Gateway is added to the architecture, in order to re-route the API calls to their specific microservice.
You need to change the URL for the API in accordance with their respective microservice URLs, in the env file present in the APIGateway folder structure.

### CODE REVIEW

File: APIGateway/src/gateway/__inti__.py
Comment (shahih25) [inline APIGateway/src/gateway/__inti__.py:1]: Filename: **__init__.py**

**Comment by shahih25** [comment]
Is it possible to have the code structure as APIGateway/controllers, APIGateway/models, APIGateway/service? Like move the content from gateway folder to APIGateway?

**Review by sverma89asu — APPROVED** [review]
LGTM

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**